package.json

```diff
@@ -387,7 +387,7 @@
       "eslint --rulesdir frontend/lint/eslint-rules --max-warnings 0 --fix",
       "prettier --write"
     ],
-    "e2e/test/scenarios/*/{*.js,!(helpers|shared)/*.js}": [
+    "e2e/test/scenarios/*/{*.(js|ts),!(helpers|shared)/*.(js|ts)}": [
```

Comment:
One minor comment, not exactly about this change, but the line below.

Given we already run every staged .ts and .js file with `eslint --rulesdir frontend/lint/eslint-rules --max-warnings 0 --fix` via line 387.

Wouldn't it be doing the same thing twice on line 391? My hunch is that we could remove line 391 and leave only `node e2e/validate-e2e-test-files.js`

Reply:
Nice catch :heart:  45bafbfd30